lib/menus/utils/image-util.js

```diff
@@ -0,0 +1,102 @@
+import { layouts } from 'minecraft-inventory-gui/lib/layouts.mjs'
```

Comment:
This file can be refactored for use the image data already bundled inside the client.